examples/prometheus/README.md

```diff
@@ -0,0 +1,211 @@
+# Getting Started with Prometheus and Grafana
+
+- [Export metrics from the application](#export-metrics-from-the-application)
+  - [Check results in the browser](#check-results-in-the-browser)
+- [Collect metrics using Prometheus](#collect-metrics-using-prometheus)
+  - [Configuration](#configuration)
+  - [Start Prometheus](#start-prometheus)
+  - [View results in Prometheus](#view-results-in-prometheus)
+- [Explore metrics using Grafana](#explore-metrics-using-grafana)
+- [Learn more](#learn-more)
+
+## Export metrics from the application
+
+It is highly recommended to go over the [ostream-metrics](../metrics_simple/README.md)
+doc before following along this document.
+
+Create a new console application and run it:
+
+```sh
+bazel run //examples/prometheus:prometheus_example
+```
+
+Now, we are going to make some small tweaks to the example in the
```

Comment:
Should we be making these changes in the example before running it?

Reply:
Sorry if I am missing something. Why do we need to modify ostream-metrics to export to Prometheus Exporter - the above step of running prometheus_example should already do the metrics export?

Reply:
thanks, it is not needed to do anything extra here. I changed the wording to make it clear.